src/bp-blogs/bp-blogs-filters.php

```diff
@@ -134,3 +134,44 @@ function bp_blogs_register_custom_site_icon_size( $sizes ) {
 	return $sizes;
 }
 add_filter( 'site_icon_image_sizes', 'bp_blogs_register_custom_site_icon_size' );
+
+/**
+ * Set view post button for activit post content.
+ *
+ * @since BuddyBoss 1.7.1
+ *
+ * @param array $buttons     Group buttons.
+ * @param int   $activity_id Activity id.
+ * 
+ * @return array
+ */
+function bb_nouveau_get_activity_inner_blogs_buttons( $buttons, $activity_id ) {
+	global $activities_template;
+
+	if ( 
+		( 'blogs' === $activities_template->activity->component ) 
+		&& 
+		isset( $activities_template->activity->secondary_item_id ) 
+		&& 
+		'new_blog_' . get_post_type( $activities_template->activity->secondary_item_id ) === $activities_template->activity->type
+	) {
+		$blog_post = get_post( $activities_template->activity->secondary_item_id );
+		// If we converted $content to an object earlier, flip it back to a string.
+		if ( is_a( $blog_post, 'WP_Post' ) && ! has_post_thumbnail( $blog_post ) ) {
+			$buttons['activity_post'] = array(
+				'id'                => 'activity_post_link_wrap',
+				'position'          => 4,
+				'component'         => 'activity',
+				'button_element'    => 'a',
+				'link_text'         => sprintf( '<span class="text">%1$s</span>', esc_html__( 'View Post', 'buddyboss' ) ),
+				'button_attr'       => array(
+					'href'          => esc_url( get_permalink( $blog_post->ID ) ),
+					'class'             => 'button bb-icon-arrow-circle bp-secondary-action',
+				),
+			);
+		}
+	}
+
+	return $buttons;
+}
```

Comment:
@asaquzzamanbb Please follow the phpcs here.

Reply:
@KartikSuthar 
Done https://github.com/buddyboss/buddyboss-platform/pull/2290/commits/ada2f3b2a8ba1617bb59852fec2eee07fdd301c5